packages/ui/components/input-amount/src/parsers.js

```diff
@@ -31,14 +31,17 @@ function round(value, decimals) {
  * @param {FormatNumberOptions} [givenOptions] Locale Options
  */
 export function parseAmount(value, givenOptions) {
-  const unmatchedInput = value.match(/[^0-9,.\- ]/g);
+  // Replace Unicode minus with normal minus before parsing
+  const sanitizedValue = value.replace(/\u2212/g, '-');
+
+  const unmatchedInput = sanitizedValue.match(/[^0-9,.\- ]/g);
```

Comment:
Inside the `parseNumber()` function we process the minus signs and set the Unicode value. So all we have to do is whitelist it here and it works. Also no need to change the formatter.

```suggestion
  const unmatchedInput = sanitizedValue.match(/[^0-9,.\-\u2212 ]/g);
```